contracts/utility/Upgradeable.sol

```diff
@@ -88,10 +88,10 @@ abstract contract Upgradeable is IUpgradeable, AccessControlEnumerableUpgradeabl
             revert AlreadyInitialized();
         } else if (!checkVersion) {
             initializations = _version;
-        } else {
-            initializations = initializations;
         }
 
+        _initializations = initializations;
```

Comment:
Nitpick: you write twice to `_initializations` (but it's also not the end of the world, since it's called only during an upgrade)

Reply:
only one write I think (we set `initializations` on line 85, then we write to it on line 90, and then we set `_initializations` to `initializations` at the end)